src/metadata-extractors.ts

```diff
@@ -71,6 +73,9 @@ function getTypeInfo(
         node.typeArguments?.map((node) => getTypeInfo(node, context)) ?? []
     } else if (ts.isFunctionTypeNode(node)) {
       typeInfo.name = 'Function' // TODO: We could get more detailed here
+    } else if (ts.isArrayTypeNode(node)) {
+      typeInfo.isArray = true
+      typeInfo.name = node.elementType.getText()
```

Comment:
In this case, we treat it as if it were `Array<...>`:

```suggestion
    } else if (ts.isArrayTypeNode(node)) {
      typeInfo.name = Array.name
      typeInfo.name = node.typeArguments?.map((node) => getTypeInfo(node, context)) ?? []
```

Reply:
`typeArguments` does not exist on `node` due to the type guard of `isArrayTypeNode`, but I think `elementType` without `.map` does the trick.